### PR TITLE
Flatten and rename SQS message

### PR DESCRIPTION
**Pull request by sudheshna-donthineni**

### CODE REVIEW

**Review by suryabulusu — APPROVED** [review]
looks good

File: lambda_function.py [hunk 1]
```diff
@@ -35,36 +37,31 @@ def lambda_handler(event, lambda_context):
...
+                row["attendance_timestamp"] = message["date_time"]
+                row["purpose_type"] = message["type"]
+                row["purpose_subtype"] = message["sub_type"]
+                row["platform"] = message["platform"]
+                row["platform_id"] = message["platform_id"]
+                row["auth_type"] = message["auth_type"]
+                row["user_id"] = message["user_id"]
+                row["user_data_validated"] = message["user_validated"]
+                row["number_of_multiple_entries"] = 1
+                row["userType"] = message["user_type"]
```
Comment (suryabulusu) [inline lambda_function.py:55]: shouldn't it be row["user_type"]

**Review by suryabulusu — COMMENTED** [review]
oops added a comment

File: lambda_function.py (hunk 1 above)
Comment (sudheshna-donthineni) [reply lambda_function.py:55]: No, unfortunately this was how the BQ column was named when we first started

**Pull request merged**